config/prod/iAtlas-large2.yaml

```diff
@@ -0,0 +1,46 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v1.yaml
```

Comment:
there's a newer version `managed-ec2-ubuntu-v2.j2`.  please use the example template here: https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/147/files